apps/squzy_storage/main.go

```diff
@@ -17,21 +18,17 @@ import (
 func main() {
 	tools := grpctools.New()
 	cfg := config.New()
-	postgresDb, err := gorm.Open(
-		"postgres",
-		fmt.Sprintf("host=%s port=%s dbname=%s user=%s  password=%s connect_timeout=10 sslmode=disable",
-			cfg.GetDbHost(),
-			cfg.GetDbPort(),
-			cfg.GetDbName(),
-			cfg.GetDbUser(),
-			cfg.GetDbPassword(),
-		))
-
+	connect, err := sql.Open("clickhouse", fmt.Sprintf("tcp://%s:%s?username=%s&password=%s&database=%s&read_timeout=10&write_timeout=20",
```

Comment:
I suppose it will be good to have an option in config to select FB type. 
Like `DbType: "postgress"/"clickhouse"`